wp-content/themes/core/blocks/tribe/terms/render.php

```diff
@@ -15,45 +15,35 @@
 
 echo '<div ' .  wp_kses_data( get_block_wrapper_attributes() ) . '>';
 
-if ( 0 === count( $terms_block_terms ) ) {
-	echo '<!-- Terms block: No terms to list. -->';
-	echo '</div>';
-
-	return;
+// No terms and we're in the block editor
+if ( 0 === count( $terms_block_terms ) && ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
```

Comment:
I think we should probably extract `( defined( 'REST_REQUEST' ) && REST_REQUEST ) )` into a utility/helper somewhere more global. It feels weird to check like this in the context. But, food for thought.